versioned_docs/version-V2/use-cases/private-voting.md

```diff
@@ -0,0 +1,135 @@
+---
+sidebar_position: 2
+---
+
+# Private voting use case
+
+The private voting use case describes how Semaphore interacts with your users and Ethereum to allow users to cast private votes in your application.
+Learn how Semaphore enables applications to do the following:
+
+- Register members as voters.
+- Allow members to vote anonymously.
+- Prove voter membership.
+- Record and prove votes.
+- Prevent double-voting.
+
+## Roles
+
+- **[Developer or community admin](#developer-or-community-admin)**
+- **[Community member (dApp user)](#community-member)**
+- **[Relay](#relay)**
+
+### Developer or community admin
+
+As a developer or community admin, you deploy the following:
+
+- **Smart contract on Ethereum**: implements the Semaphore **base contract** to post transactions and verify proofs on Ethereum.
+- **Poll**: Semaphore _group_ that members join to vote on proposals.
```

Comment:
Polls (groups) should be part of the previews smart contract. **base contracts** means `SemaphoreCore` and `SemaphoreGroups`

Reply:
Fixed.